### PR TITLE
TheNOP.io Pledge

### PR DESCRIPTION
**Pull request by EmreNOP**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
 # Agoric Validator Pledge
 
-_context: see Jan 28 [Validator Delegation Program](https://agoric.com/blog/announcements/agoric-validator-program/) announcement._
+I,TheNOP.io hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```
Comment (jeetraut) [inline README.md:3]:
```suggestion
[TheNop.io Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper17k4wstnvtyt7pt8qnfamrdpkkj7f5qqvf6heg3)
I,TheNOP.io hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Review by EmreNOP — COMMENTED** [review]
ok

**Comment by EmreNOP** [comment]
Video about, Keplr installation, coinlist process and delegation for Turkish community

https://www.youtube.com/watch?v=rO0hzWXOZ8I

**Review by jeetraut — APPROVED** [review]

**Pull request merged**